src/test/java/dk/sdu/se_f22/sortingmodule/range/rangepublic/EqualsTest.java

```diff
@@ -0,0 +1,367 @@
+            @CsvFileSource(resources = "TimeFilter.csv", numLinesToSkip = 1)
+            void timeFiltersWithDifferentFilterTypesDoubleFilter(int id, String name, String description, String productAttribute, Instant min, Instant max) {
+                TimeFilter timeFilter = new TimeFilter(id, name, description, productAttribute, min, max);
+                DoubleFilter doubleFilterWithDifferentType = new DoubleFilter(id, name, description, productAttribute, 0.1, 99.9);
+                assertNotEquals(timeFilter,doubleFilterWithDifferentType);
+            }
+            @ParameterizedTest(name = "Double filter: {0},{1},{2},{3}, 1, 100")
+            @DisplayName("Time filters with different filter types: Long filter")
+            @CsvFileSource(resources = "TimeFilter.csv", numLinesToSkip = 1)
+            void timeFiltersWithDifferentFilterTypesLongFilter(int id, String name, String description, String productAttribute, Instant min, Instant max) {
+                TimeFilter timeFilter = new TimeFilter(id, name, description, productAttribute, min, max);
+                DoubleFilter doubleFilterWithDifferentType = new DoubleFilter(id, name, description, productAttribute, 1, 100);
+                assertNotEquals(timeFilter,doubleFilterWithDifferentType);
+            }
+
+            @ParameterizedTest(name = "Different user min: {0},{1},{2},{3},{4},{5}")
+            @DisplayName("Time filters with different UserMin values")
+            @CsvFileSource(resources = "TimeFilter.csv", numLinesToSkip = 1)
+            void timeFiltersWithDifferentUserMinValues(int id, String name, String description, String productAttribute, Instant min, Instant max) throws IllegalMinMaxException {
+                TimeFilter timeFilter = new TimeFilter(id, name, description, productAttribute, min, max);
+                TimeFilter timeFilterWithDifferentUserMin = new TimeFilter(id, name, description, productAttribute, min, max);
+                timeFilterWithDifferentUserMin.setUserMin(min);
+                assertNotEquals(timeFilter,timeFilterWithDifferentUserMin);
+            }
+
+            @ParameterizedTest(name = "Different user max: {0},{1},{2},{3},{4},{5} ")
+            @DisplayName("Time filters with different UserMax values")
+            @CsvFileSource(resources = "TimeFilter.csv", numLinesToSkip = 1)
+            void timeFiltersWithDifferentUserMaxValues(int id, String name, String description, String productAttribute, Instant min, Instant max) throws IllegalMinMaxException {
+                TimeFilter timeFilter = new TimeFilter(id, name, description, productAttribute, min, max);
+                TimeFilter timeFilterWithDifferentUserMax = new TimeFilter(id, name, description, productAttribute, min, max);
+                timeFilterWithDifferentUserMax.setUserMax(max);
+                assertNotEquals(timeFilter,timeFilterWithDifferentUserMax);
+            }
+        }
+    }
+
+    @Nested
+    @DisplayName("Matching tests")
```

Comment:
Same naming concern although it would be fair to take it down to the individual tests (although the best thing would be to communicate it both places

Reply:
Done